yaml/parse.ts

```diff
@@ -14,7 +14,7 @@ export interface ParseOptions {
   /** Name of the schema to use.*/
   schema?: "core" | "default" | "failsafe" | "json" | "extended";
   /** compatibility with JSON.parse behaviour. */
```

Comment:
Let's also change the description

Reply:
I'll be reworking docs for all of `@std/yaml` once the breaking changes are made.